jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartFormData.java

```diff
@@ -450,17 +461,17 @@ private class PartsListener extends MultiPart.AbstractPartsListener
             public void onPartContent(Content.Chunk chunk)
             {
                 ByteBuffer buffer = chunk.getByteBuffer();
+                long maxFileSize = getMaxFileSize();
+                fileSize += buffer.remaining();
```

Comment:
tiny change for readability:
```suggestion
                fileSize += buffer.remaining();
                long maxFileSize = getMaxFileSize();
```